imod/mf6/hfb.py

```diff
@@ -343,6 +345,8 @@ def to_mf6_pkg(
 
         Parameters
         ----------
+        validate: bool
+            Run validation before converting
```

Comment:
Please put the arguments in the docstring in the same order as in the actual code.